docs/src/man/hosting.md

```diff
@@ -86,13 +86,18 @@ Linux. This can be changed using the `julia` and `osname` keywords as follows:
 
 ```julia
 deploydocs(
+    deps   = Deps.pip("python-markdown-math"),
```

Comment:
This will only install `python-markdown-math` and not `mkdocs`, which means it'll fail to deploy and could be confusing to readers. Probably best to describe the `deps` options and `mdx_math` in a separate example after this one. Perhaps directly after the

```
This will deploy the docs from the OSX Julia 0.4 Travis build bot.
``` 

line?